tmail-backend/apps/memory/src/main/java/com/linagora/tmail/james/app/NoopGuiceLoader.java

```diff
@@ -0,0 +1,114 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+package com.linagora.tmail.james.app;
+
+import java.util.stream.Stream;
+
+import org.apache.james.utils.ClassName;
+import org.apache.james.utils.FullyQualifiedClassName;
+import org.apache.james.utils.GuiceLoader;
+import org.apache.james.utils.NamingScheme;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.collect.ImmutableList;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import com.google.inject.Module;
+
+public class NoopGuiceLoader implements GuiceLoader {
```

Comment:
Relocate this class in https://github.com/linagora/tmail-backend/tree/master/tmail-backend/guice/common/src/main/java/com/linagora/tmail ?